packages/salesforcedx-vscode-apex-debugger/.vscodeignore

```diff
@@ -16,12 +16,13 @@ node_modules/@salesforce/salesforcedx-apex-debugger/package-lock.json
 node_modules/@salesforce/salesforcedx-apex-debugger/tsconfig.json
 
 # salesforcedx-utils-vscode files to ignore 
-#node_modules/@salesforce/salesforcedx-apex-debugger/node_modules/@salesforce/salesforcedx-utils-vscode/node_modules/**
+node_modules/@salesforce/salesforcedx-apex-debugger/node_modules/@salesforce/salesforcedx-utils-vscode/node_modules/**
```

Comment:
Ok maybe silly question, but why did these files get to stay in the `.vscodeignore`, but you seemed to remove them from most of the other packages?

Reply:
mostly b/c I got tired.  There's def a lot more cleanup that can be done in the vscodeignore files.  And some basic rules that make it easy to know what needs to be in here.  These apex-debugger lines can 100% be removed b/c that module is part of the bundling so it's not even present in the extension directory. Good catch.